### PR TITLE
chore: Remove commit body and PR# from post-processor-changes-txt

### LINKED ISSUE #1134: Remove the relative issue number in post-processor-changes.txt
#1120 added `post-processor-changes.txt` to the [cloudbuild.yaml](https://github.com/googleapis/synthtool/blob/master/docker/owlbot/python/cloudbuild.yaml#L7) file for python in order to have the latest commit message available to the `owlbot-python` post processor image. The relative issue number should be removed from `post-processor-changes.txt` as the contents of the file will land in downstream repositories . As an example for [this commit](https://github.com/googleapis/synthtool/commit/2514a9985b36626acb384dff903c62ac5963441b), `post-processor-changes.txt` should contain the following
```
chore(python): remove hardcoded versions in CONTRIBUTING.rst (#1132)
```
instead of
```
chore(python): remove hardcoded versions in CONTRIBUTING.rst
```

### PR DESCRIPTION
**Pull request by parthea**
Fixes #1134 🦕

Removes the commit body and relative PR# from the commit message.

For example, for this commit: https://github.com/googleapis/synthtool/commit/9763f20e4b7bb1091082462b2f7970e965d0d414

`post-processor-changes.txt` would contain

```
build: enable npm for php/python builds 

Source-Link: https://github.com/googleapis/synthtool/commit/9763f20e4b7bb1091082462b2f7970e965d0d414
```

instead of

```
build: enable npm for php/python builds (#1133)

* build: enable npm for php/python builds

* update comment
Source-Link: https://github.com/googleapis/synthtool/commit/9763f20e4b7bb1091082462b2f7970e965d0d414
```

### CODE REVIEW

File: docker/owlbot/java/cloudbuild.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ steps:
     entrypoint: '/bin/sh'
     args:
       - '-c'
-      - 'git log -1 --format="%s%n%n%b%nSource-Link: https://github.com/googleapis/synthtool/commit/%H" > post-processor-changes.txt'
```
Comment (SurferJeffAtGoogle) [inline docker/owlbot/java/cloudbuild.yaml:20]: I haven't heard the same request from Java or NodeJs people, so please make this change to python's cloudbuild.yaml only.

File: docker/owlbot/python/cloudbuild.yaml [hunk 2]
```diff
@@ -4,7 +4,7 @@ steps:
     entrypoint: '/bin/sh'
     args:
       - '-c'
-      - 'git log -1 --format="%s%n%n%b%nSource-Link: https://github.com/googleapis/synthtool/commit/%H" > post-processor-changes.txt'
+      - 'git log -1 --format="%s%n%nSource-Link: https://github.com/googleapis/synthtool/commit/%H" | sed -e "s/([^()]*)//g" > post-processor-changes.txt'
```
Comment (SurferJeffAtGoogle) [inline docker/owlbot/python/cloudbuild.yaml:7]: This sed statement will remove all parenthesized text:
```
$ echo "Hello (Good Morning) World (#123)" | sed -e "s/([^()]*)//g"
Hello  World
```

Is that acceptable for your purposes?

File: docker/owlbot/java/cloudbuild.yaml (hunk 1 above)
Comment (parthea) [reply docker/owlbot/java/cloudbuild.yaml]: Done.

File: docker/owlbot/python/cloudbuild.yaml (hunk 2 above)
Comment (parthea) [reply docker/owlbot/python/cloudbuild.yaml]: Good catch! I only wanted to remove the parenthesized text at the end. Fixed.

**Comment by parthea** [comment]
This is ready for review.

**Comment by parthea** [comment]
@SurferJeffAtGoogle PTAL

**Review by SurferJeffAtGoogle — APPROVED** [review]

**Pull request merged**